src/main/java/sirius/biz/codelists/LookupTable.java

```diff
@@ -731,4 +730,22 @@ public boolean canScan() {
     }
 
     protected abstract Stream<LookupTableEntry> performScan(String lang);
+
+    /**
+     * Enumerates all entries matching the given lookup in the table using the given language.
+     *
+     * @param lang        the language to translate the name and description to
+     * @param lookupPath  the field to search in
+     * @param lookupValue the value to search for
+     * @return a stream of all entries matching the lookup or an empty stream if scanning isn't supported
+     */
+    public Stream<LookupTableEntry> lookupScan(String lang, String lookupPath, String lookupValue) {
+        if (!canScan()) {
```

Comment:
IMHO this check can be skipped. "canScan" is rather a "is it smart to scan" as some lists are very big and should not be output completeley. However, as a filter is applied here, we can (hopefully) safely always perform a query